DOCS/man/mpv.rst

```diff
@@ -1501,6 +1501,30 @@ For Windows-specifics, see `FILES ON WINDOWS`_ section.
     in default configuration will use ``/usr/local/etc/mpv/`` as config
     directory, while most Linux distributions will set it to ``/etc/mpv/``).
 
+``~/.config/mpv``
+    The standard configuration directory. This can be overridden by environment
+    variables:
+
+    If ``$XDG_CONFIG_HOME`` is set, then the derived configuration directory
+    will be ``$XDG_CONFIG_HOME/mpv``.
+
+    If ``$HOME`` is set, then the derived configuration directory will be
+    ``$HOME/.config/mpv``.
+
+    If ``$MPV_HOME`` is set, then the derived configuration directory will be
```

Comment:
Isn't MPV_HOME preferred over all others? Is this list even ordered by precedence?